sdk/easm/defender-easm-rest/review/defender-easm.api.md

```diff
@@ -683,7 +714,7 @@ export interface DiscoTemplateOutput {
 }
 
 // @public (undocumented)
-export interface DomainAssetOutput extends InventoryAssetOutput {
+export interface DomainAssetOutput {
```

Comment:
what is causing this change here? It maybe a non breaking since InventoryAssetOutput is an empty object?

Reply:
need use latest main to refresh as it has enum issue in the released commit